[2021-10-06 16:20:45,556] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Pipeline.clear_cooking 2021-10-06T16:20:39.977699+00:00 [queued]>
[2021-10-06 16:20:45,567] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Pipeline.clear_cooking 2021-10-06T16:20:39.977699+00:00 [queued]>
[2021-10-06 16:20:45,568] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-10-06 16:20:45,568] {taskinstance.py:1068} INFO - Starting attempt 1 of 2
[2021-10-06 16:20:45,568] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-10-06 16:20:45,582] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): clear_cooking> on 2021-10-06T16:20:39.977699+00:00
[2021-10-06 16:20:45,589] {standard_task_runner.py:52} INFO - Started process 166 to run task
[2021-10-06 16:20:45,593] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Pipeline', 'clear_cooking', '2021-10-06T16:20:39.977699+00:00', '--job-id', '4', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/P.py', '--cfg-path', '/tmp/tmpo29_4o01', '--error-file', '/tmp/tmpk4j7rk5n']
[2021-10-06 16:20:45,594] {standard_task_runner.py:77} INFO - Job 4: Subtask clear_cooking
[2021-10-06 16:20:45,634] {logging_mixin.py:104} INFO - Running <TaskInstance: Pipeline.clear_cooking 2021-10-06T16:20:39.977699+00:00 [running]> on host 8a718b72cb7e
[2021-10-06 16:20:45,699] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=datath
AIRFLOW_CTX_DAG_ID=Pipeline
AIRFLOW_CTX_TASK_ID=clear_cooking
AIRFLOW_CTX_EXECUTION_DATE=2021-10-06T16:20:39.977699+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-06T16:20:39.977699+00:00
[2021-10-06 16:20:45,851] {python.py:151} INFO - Done. Returned value was: None
[2021-10-06 16:20:45,862] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=Pipeline, task_id=clear_cooking, execution_date=20211006T162039, start_date=20211006T162045, end_date=20211006T162045
[2021-10-06 16:20:45,886] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-10-06 16:20:45,925] {local_task_job.py:151} INFO - Task exited with return code 0
